style.css

```
@@ -0,0 +1,58 @@
+@font-face{
+    font-family: Acrobat;
+    src: url(Akrobat-Bold.otf);
+}
+body{
+    font-family: Acrobat;

```

W razie jakby nie mógł pobrać czcionki
```suggestion
    font-family: Acrobat, sans-serif;
```